services/node_create.proto

```diff
@@ -125,4 +127,14 @@ message NodeCreateTransactionBody {
      * This field is OPTIONAL.
      */
     bytes grpc_certificate_hash = 6;
+
```

Comment:
The package name of this file is wrong `package com.hedera.hapi.node.addressbook;` It should just be `proto` similar to other services.

Reply:
As discussed in slack, `package com.hedera.hapi.node.addressbook;` is (or should be) correct.  We'll confirm that there are no issues with metrics, but barring that we should keep the more complete package directive.